src/functions-reference/matrix_operations.Rmd

```diff
@@ -869,6 +869,37 @@ The sample standard deviation of the values in x; see section
 The sample standard deviation of the values in x; see section
 [array reductions](#array-reductions) for details.
 
+### Quantile
+
+Produces sample quantiles corresponding to the given probabilities.
+The smallest observation corresponds to a probability of 0 and the largest
+to a probability of 1.
+
+<!-- real; quantile; (data vector, data real p); -->
+\index{{\tt \bfseries quantile }!{\tt (data real[] x, data real p): real}|hyperpage}
+
+`real` **`quantile`**`(data vector, data real p)`<br>\newline
+The p-th quantile of xs.
+
+<!-- real[]; quantile; (data vector, data real[] p); -->
+\index{{\tt \bfseries quantile }!{\tt (data real[] x, data real[] p): real}|hyperpage}
+
+`real[]` **`quantile`**`(data vector, data real ps[])`<br>\newline
+An array containing the p-th quantile of xs for each p in ps.
```

Comment:
If you change `ps` to `p`, then the end of this sentence would be something like, `An array containing the quantiles of x given by the array p`